.gitignore

```
@@ -170,3 +170,8 @@ Xcode/
 
 # gdb
 .gdb_history
+
+# Flatpak build artifacts
+/.flatpak-builder/
+/repo/
+/build-dir/
```

```suggestion
/build-dir/

```

You need an empty line as last line of any file.